python/lsst/ts/salobj/base_csc.py

```diff
@@ -183,7 +177,15 @@ def __init__(
                     f"simulation_mode={simulation_mode} "
                     f"not in valid_simulation_modes={self.valid_simulation_modes}"
                 )
-        self.evt_simulationMode.set(mode=int(simulation_mode))
+
+        self._requested_simulation_mode = int(simulation_mode)
```

Comment:
If this variable is gone, why is this reference here? Is this only used as a safety operator? Found another reference in the start function.

Reply:
Good catch. I've removed them. A global search showed no use in other packages (and it is a private variable).